### PR TITLE
Update to CDAP 3.1.0 release

### PR DESCRIPTION
**Pull request by wolf31o2**

### CODE REVIEW

**Comment by dereklwood** [comment]
Lgtm

File: attributes/sdk.rb [hunk 1]
```diff
@@ -35,6 +35,8 @@
     '67f6e93b47591ea7bdbc7e66416f9858ab0c75eb3c9c338bf109b81089310e52'
   when '3.0.3'
     '54a4cc4ef0a0148a97b571de5284660b393f2277eea86c70a94d8a83208187b1'
+  when '3.1.0'
+    '96ac34729363d8926b3a44fe6a70db5671e9e6bcde38651304352e3d48d35cde'
```
Comment (sreevatsanraman) [inline attributes/sdk.rb]: We will have another SDK build. Please keep this PR open. We can update the Shasum after the new build.

Comment (sreevatsanraman) [inline attributes/sdk.rb]: New shasum 0b868aafcd0801e35ab52ead70537a63ec0eb5c458c6e7c3ace363a8fd4d48d2
`shasum -a 256 cdap-sdk-3.1.0.zip`
`0b868aafcd0801e35ab52ead70537a63ec0eb5c458c6e7c3ace363a8fd4d48d2  cdap-sdk-3.1.0.zip`

**Comment by sreevatsanraman** [comment]
LGTM

**Pull request merged**